woocommerce/woocommerce-ios
# Aztec editor for Product description: part 2.3 - link format action

Part 2.3 for #1421

## Changes
- The behavior and setup are pretty similar to the version on WPiOS
- Created `SettingTitleAndValueTableViewCell` that displays a setting title and optional value
- Created `LinkSettingsViewController` that displays the fields for the user to configure the link settings
  - Showed the URL and text cells with `SettingTitleAndValueTableViewCell`
- Added `AztecLinkFormatBarCommand` that presents `LinkSettingsViewController` to format the link settings of the text in Aztec
- Updated `TextViewViewController`:
  - Added a configuration for keyboard type
  - Brought up the keyboard in `viewWillAppear` 

## Testing
- Launch the app
- Go to Products tab
- Tap on a Product
- Tap on the description row
- In the editor, play with the "🔗" format action in the keyboard toolbar --> try adding a link, and the link text, URL and whether to open in a new window should be correctly updated (you can verify them in the source code mode as well)
- Try removing the link after adding it to the text

## Example screenshots

Link Settings dialog | Edit URL | Edit link text
-- | -- | --
![Simulator Screen Shot - iPhone 8 - 2019-11-20 at 18 08 10](https://user-images.githubusercontent.com/1945542/69232990-0811c080-0bc7-11ea-83ce-d72fd9b010bb.png) | ![Simulator Screen Shot - iPhone 8 - 2019-11-20 at 18 53 57](https://user-images.githubusercontent.com/1945542/69233055-2677bc00-0bc7-11ea-8cdc-0e313dd8d8c1.png) | ![Simulator Screen Shot - iPhone 8 - 2019-11-20 at 18 16 31](https://user-images.githubusercontent.com/1945542/69232991-08aa5700-0bc7-11ea-9663-754ecead57ad.png)

Update release notes:
